foodDeliveryProject/description.txt

```diff
@@ -0,0 +1,176 @@
+Admin {
```

Comment:
add this file in gitignore no need to push this to github.

Reply:
noted